GiEnJul/Controllers/AdminController.cs

```diff
@@ -41,18 +43,21 @@ public AdminController(IEventRepository eventRepository, IGiverRepository giverR
         // }
 
         [HttpGet("givers")]
+        // [Authorize] This will deny any unauthorized requests, but will break the application atm
+        [Authorize(Policy = "ReadGivers")]
```

Comment:
Is line 46 still true?

Reply:
Changed this to //[Authorize(Policy = "ReadGivers"] This will deny any unauthorized requests, but will break the application atm
So Line 46 is still true. Waiting on the change for adminpage from #250 first, so it can be implemented

Reply:
@Sommerli